USAGEGUIDE.md

```diff
@@ -118,6 +125,64 @@ make deploy
 make undeploy
 ```
 
+## Kubernetes Operator Deployment with Service Account
```

Comment:
Currently users would need manually to modify `manager.yaml` to set Service Account token. There is a followup ticket to make connect/service account setup automatic https://github.com/1Password/onepassword-operator/issues/200